src/client/wire_protocol/header.rs

```diff
@@ -102,20 +107,54 @@ impl Header {
         Header::new(message_length, request_id, 0, op_code)
     }
 
+    /// Construcs a new Header for a reply.
+    ///
+    /// # Arguments
+    ///
+    /// `message_length` - The length of the message in bytes.
+    /// `request_id` - Identifier for the request, or `0` if the the message
```

Comment:
I was just following the example in the style guideline of lining up each line of a single argument but not the arguments themselves: https://github.com/rust-lang/rust/blob/master/src/doc/style/style/comments.md